src/ethereum/ethNetwork.js

```diff
@@ -1447,451 +1265,6 @@ export class EthereumNetwork {
     }
   }
 
-  async checkTxsAlethio(
```

Comment:
I'm less picky about this being removed (again) because I don't think the service even exists anymore.